.github/CODEOWNERS

```diff
@@ -3,4 +3,3 @@
 
 # Changes to `fetch-configlet` should be made in the `exercism/configlet` repo
 bin/fetch-configlet     @exercism/maintainers-admin
-
```

Comment:
Could you re-add the trailing newline? They make diffs easier

Reply:
I agree.  The blank line (but without trailing space) is helpful, indeed.